src/app/app.component.scss

```diff
@@ -0,0 +1,134 @@
+$width280: 280px;
```

Comment:
global scss file is not working in angular app.there may be one solution to import one scss where we declare all variable in every component.